addons/account_voucher/views/account_voucher_views.xml

```diff
@@ -269,7 +274,7 @@
                         </page>
                         <page string="Journal Items">
                             <group>
-                                <field name="account_date"/>
+                                <field name="account_date" groups="base.group_no_one"/>
```

Comment:
mmh are you sure?? I'd let that visible by default: we're on the second page anyway

Reply:
requested by Laura... see
https://drive.google.com/file/d/1ZX-_6kgZMBl42M-kiqnWZeQnUO6Aw7fy/view

Reply:
it seems that the screenshot doesn't relate to this view: you are modifying a field in a notebook's second page, and the screenshot shows a field outside a notebook

Reply:
I spoke to Laura! She said "We don't care about the accounting date in 10.0"! But these changes are correct in master